ui/component/or-dashboard-builder/src/settings/attribute-settings.ts

```diff
@@ -0,0 +1,620 @@
+    color: #6f6f6f;
+  }
+
+  .dropdown-item:hover,
+  .dropdown-item:focus {
+    background: #f5f5f8;
+    outline: none;
+  }
+`;
+
+@customElement("attribute-settings")
+export class AttributeSettings extends AssetWidgetSettings {
+  protected readonly widgetConfig!: AttributeWidgetConfig;
+  @property({ type: Boolean }) private showMenu = false;
+
+  @state() private rangeErrors: Record<number, string> = {};
+
+  static get styles() {
+    return [...super.styles, styling];
+  }
+
+  protected render(): TemplateResult {
+    return html`
+      <div>
+        <!-- Attribute selection -->
+        <settings-panel displayName="attributes" expanded="true">
+          <attributes-panel
+            .attributeRefs=${this.widgetConfig.attributeRefs}
+            style="padding-bottom: 12px;"
+            @attribute-select=${(ev: AttributesSelectEvent) => this.onAttributesSelect(ev)}
+          ></attributes-panel>
+          <div class="field">
+            <or-mwc-input
+              .type=${InputType.TEXT}
+              .value=${this.widgetConfig.variableLabel ?? ""}
+              @or-mwc-input-changed=${(e: OrInputChangedEvent) => this.onVariableLabelChange(e)}
+              label="Neuer Name der Variable"
+            ></or-mwc-input>
+          </div>
+        </settings-panel>
+
+        <!-- Icon visibility toggle -->
+        <settings-panel displayName="Icon Einstellung" expanded="true">
+          <div class="field">
+            <span>Icon anzeigen</span>
+            <or-mwc-input
+              .type=${InputType.SWITCH}
+              .value=${this.widgetConfig.showIcon}
+              @or-mwc-input-changed=${(ev: OrInputChangedEvent) => this.onShowIconToggle(ev)}
+            ></or-mwc-input>
+          </div>
+
+          ${this.widgetConfig.showIcon
+            ? html`
+                <div class="field">
+                  <span>Basisfarbe: </span>
+                  <or-mwc-input
+                    type="${InputType.COLOUR}"
+                    value="${this.widgetConfig.iconColor}"
+                    @or-mwc-input-changed="${(event: Event) => this.onIconColorChange(event)}"
+                  ></or-mwc-input>
+                </div>
+
+                <div class="field">
+                  <span>Icon auswählen</span>
+                  <div class="icon-picker">
+                    ${ICON_OPTIONS.map(
+                      (iconName) => html`
+                        <div
+                          class="icon-option"
+                          ?selected=${this.widgetConfig.icon === iconName}
+                          @click=${() => this.onIconPicked(iconName)}
+                        >
+                          <or-icon icon=${iconName} style="--or-icon-width:32px; --or-icon-height:32px;"></or-icon>
+                        </div>
+                      `
+                    )}
+                  </div>
+                </div>
+              `
+            : null}
+        </settings-panel>
```

Comment:
This section should be aligned differently.
Similar to the other widgets, I would align texts to the left, and controls to the right.
I would also vertically align them, so the text is at the same height as the controls.